clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -260,7 +260,11 @@ export class CopybookDownloadService implements vscode.Disposable {
         // Can be exposed later on as a setting.
         newContent = newContent.replace(/.{80}/g, `$&\n`);
       }
-      fs.writeFileSync(filePath, newContent);
+      await fs.writeFile(filePath, newContent, (err) => {
```

Comment:
This does not do what you think... use `fs.promises.writeFile` or construct the promise yourself.